release-notes/6.0/cve.md

```diff
@@ -0,0 +1,27 @@
+# .NET 6 CVEs
+
+The .NET Team releases [monthly updates for .NET 6](https://github.com/dotnet/announcements/labels/.NET%206.0), on Update Tuesday. These updates often include security fixes. If you are on an older version, your app my be vulnerable.
```

Comment:
Should we call it "Patch Tuesday" or is "Update Tuesday" more industry-accepted?

Reply:
If there's an wikipedia article about it, it should exist 😉 
https://en.wikipedia.org/wiki/Patch_Tuesday

I think Patch Tuesday is the terminology we have been using including in the website

Reply:
We have been using "Release day" rather than "patch Tuesday" terminology since it is possible (although unlikely) we may not have a release on a Patch Tuesday

Reply:
The point about talking about Patch Tuesday is that we don't ship on another day of the month, even if we don't ship that month.

I'll update the term.